.travis.yml

```diff
@@ -37,8 +51,17 @@ before_install:
 install: true
 script:
     - cd boost-common
-    - travis_wait mvn install -Dinvoker.streamLogs=true
+    - mvn install -Dinvoker.streamLogs=true
     - cd ../boost-maven
-    - travis_wait mvn install -Pit -Dinvoker.streamLogs=true -Druntime=$RUNTIME -DruntimeVersion=$RUNTIME_VERSION
-    - cd ../boost-gradle
-    - travis_wait ./gradlew install check -Druntime=$RUNTIME -DruntimeVersion=$RUNTIME_VERSION --stacktrace --info --no-daemon
+    - mvn install -Dinvoker.streamLogs=true
```

Comment:
Why does the `boost-maven` part run unconditionally?   Is it because you're thinking ahead to bom/booster use cases?

Reply:
Yup.